views/index.html

```diff
@@ -2,7 +2,7 @@
 
 {% include "macros.html" %}
 
-{% block title %}{{ gettext("Gallery") }}{% endblock %}
```

Comment:
Fails `grunt validate` because `Gallery` is still in the locale strings. Can probably just remove it.